sdk/keyvault/azure-keyvault-securitydomain/azure/keyvault/securitydomain/_patch.py

```diff
@@ -0,0 +1,270 @@
+        :return: An instance of LROPoller that returns SecurityDomainObject. The
+         SecurityDomainObject is compatible with MutableMapping
+        :rtype:
+         ~azure.core.polling.LROPoller[~azure.keyvault.securitydomain.models.SecurityDomainObject]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+    @overload
+    def begin_download(
+        self, certificate_info_object: IO[bytes], *, content_type: str = "application/json", **kwargs: Any
+    ) -> LROPoller[SecurityDomainObject]:
+        """Retrieves the Security Domain from the managed HSM. Calling this endpoint can
+        be used to activate a provisioned managed HSM resource.
+
+        :param certificate_info_object: The Security Domain download operation requires customer to
+         provide N certificates (minimum 3 and maximum 10)
+         containing a public key in JWK format. Required.
+        :type certificate_info_object: IO[bytes]
+        :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
+         Default value is "application/json".
+        :paramtype content_type: str
+        :return: An instance of LROPoller that returns SecurityDomainObject. The
+         SecurityDomainObject is compatible with MutableMapping
+        :rtype:
+         ~azure.core.polling.LROPoller[~azure.keyvault.securitydomain.models.SecurityDomainObject]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+    @distributed_trace
+    def begin_download(
+        self, certificate_info_object: Union[CertificateInfoObject, JSON, IO[bytes]], **kwargs: Any
+    ) -> LROPoller[SecurityDomainObject]:
+        """Retrieves the Security Domain from the managed HSM. Calling this endpoint can
+        be used to activate a provisioned managed HSM resource.
+
+        :param certificate_info_object: The Security Domain download operation requires customer to
+         provide N certificates (minimum 3 and maximum 10)
+         containing a public key in JWK format. Is one of the following types: CertificateInfoObject,
+         JSON, IO[bytes] Required.
+        :type certificate_info_object: ~azure.keyvault.securitydomain.models.CertificateInfoObject or
+         JSON or IO[bytes]
+        :return: An instance of LROPoller that returns SecurityDomainObject. The
+         SecurityDomainObject is compatible with MutableMapping
+        :rtype:
+         ~azure.core.polling.LROPoller[~azure.keyvault.securitydomain.models.SecurityDomainObject]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        delay = kwargs.pop("polling_interval", self._config.polling_interval)
+        return super().begin_download(  # type: ignore[return-value]
+            certificate_info_object,
+            polling=SecurityDomainClientDownloadPollingMethod(
+                lro_algorithms=[SecurityDomainClientDownloadPolling()], timeout=delay
+            ),
+            **kwargs,
+        )
```

Comment:
The hard-coded `polling=SecurityDomainClientDownloadPollingMethod(...)` will prevent CLI from bypassing LRO with `polling=False`